.github/workflows/continuous-integration.yml

```diff
@@ -24,15 +31,38 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Cache dependencies
+      - name: Cache dependencies (Linux)
+        if: startsWith(runner.os, 'Linux')
         uses: actions/cache@v2
         with:
           path: ~/.cache/pip
           # Cache based on OS, Python version, and dependency hash
-          key: pip-${{ runner.os }}-python${{ matrix.python-version }}-${{ hashFiles('requirements-dev.txt') }}
+          key: test-${{ runner.os }}-python${{ matrix.python-version }}-${{ hashFiles('requirements-test.txt') }}
+      
+      - name: Cache dependencies (macOS)
+        if: startsWith(runner.os, 'macOS')
+        uses: actions/cache@v2
+        with:
+          path: ~/Library/Caches/pip
+          # Cache based on OS, Python version, and dependency hash
+          key: test-${{ runner.os }}-python${{ matrix.python-version }}-${{ hashFiles('requirements-test.txt') }}
+          
+      - name: Cache dependencies (Windows)
+        if: startsWith(runner.os, 'Windows')
+        uses: actions/cache@v2
+        with:
+          path: ~\AppData\Local\pip\Cache
+          # Cache based on OS, Python version, and dependency hash
+          key: test-${{ runner.os }}-python${{ matrix.python-version }}-${{ hashFiles('requirements-test.txt') }}
+      
+      - name: Install dependencies
+        run: |
+          pip install -r requirements-test.txt
+          pip install -e ".[validation]"
 
-      - name: Execute linters and test suites
-        run: ./scripts/cibuild
+      - name: Execute test suite
+        run: ./scripts/test
+        shell: bash
```

Comment:
The script already has a Bash shebang line, so this should be unnecessary.

Reply:
That's what I thought, too, but leaving this out results in the script not being run at all and that job appearing to succeed. Looking at the logs for the Windows runners, it looks like they execute the script using the following command, which means the shebang doesn't get used:

```
C:\Program Files\PowerShell\7\pwsh.EXE -command ". '{0}'"
```